source/iostreams.tex

```diff
@@ -16277,8 +16278,8 @@
 
 \pnum
 \begin{note}
-A postcondition of \tcode{last_write_time(p) == new_time} is not specified since it might not hold for file systems
-  with coarse time granularity.
+A postcondition of \tcode{last_write_time(p) == new_time} is not specified
+since it does not necessarily hold for file systems with coarse time granularity.
```

Comment:
```suggestion
because it does not necessarily hold for file systems with coarse time granularity.
```
(Drive-by: use "since" only when dealing with time, and "because" for causality.)

Reply:
fixed